diff --git a/node_modules/prettier-plugin-glsl/lib/prettier-plugin.cjs.js b/node_modules/prettier-plugin-glsl/lib/prettier-plugin.cjs.js
index f10c7e2..0cfa955 100644
--- a/node_modules/prettier-plugin-glsl/lib/prettier-plugin.cjs.js
+++ b/node_modules/prettier-plugin-glsl/lib/prettier-plugin.cjs.js
@@ -6,7 +6,6 @@ var prettier = require('prettier');
 var doc = require('prettier/doc');
 var lodash = require('lodash');
 var chevrotain = require('chevrotain');
-require('colors');
 
 function _interopNamespace(e) {
     if (e && e.__esModule) return e;
@@ -289,7 +288,7 @@ function isNode(x) {
 }
 
 const DEV = process.env.NODE_ENV !== "production";
-function underline(str, start, end, ff = (x) => x.yellow.underline, ff2 = (x) => x) {
+function underline(str, start, end, ff = (x) => x, ff2 = (x) => x) {
     return (ff2(str.substring(0, start)) +
         ff(str.substring(start, end)) +
         ff2(str.substring(end)));
@@ -308,9 +307,9 @@ function substrContext(input, token) {
             .slice(sLine - 2, eLine + 2)
             .map(([n, l]) => {
             if (n >= sLine && n <= eLine) {
-                l = underline(l, sLine === n ? token.startColumn - 1 : 0, eLine === n ? token.endColumn : l.length, (s) => s.red.underline);
+                l = underline(l, sLine === n ? token.startColumn - 1 : 0, eLine === n ? token.endColumn : l.length, (s) => s);
             }
-            return ("" + n).padStart(5).green + " " + l;
+            return ("" + n).padStart(5) + " " + l;
         })
             .join("\n"));
     // start -= 20
diff --git a/node_modules/prettier-plugin-glsl/src/checker.ts b/node_modules/prettier-plugin-glsl/src/checker.ts
index 0ffe799..68c723f 100644
--- a/node_modules/prettier-plugin-glsl/src/checker.ts
+++ b/node_modules/prettier-plugin-glsl/src/checker.ts
@@ -3,7 +3,6 @@ import * as path from "path"
 import { readFileSync } from "fs"
 import { last, pick } from "lodash"
 import { TokenType } from "chevrotain"
-import "colors"
 
 import {
   AbstractVisitor,
diff --git a/node_modules/prettier-plugin-glsl/src/util.ts b/node_modules/prettier-plugin-glsl/src/util.ts
index 6e9829e..69fbcd0 100644
--- a/node_modules/prettier-plugin-glsl/src/util.ts
+++ b/node_modules/prettier-plugin-glsl/src/util.ts
@@ -1,5 +1,4 @@
 import { Many } from "lodash"
-import "colors"
 import { isToken, Node, Token } from "./nodes"
 
 export const DEV = process.env.NODE_ENV !== "production"
@@ -14,7 +13,7 @@ export function underline(
   str: string,
   start: number,
   end: number,
-  ff = (x: string) => x.yellow.underline,
+  ff = (x: string) => x,
   ff2 = (x: string) => x,
 ): string {
   return (
@@ -91,11 +90,11 @@ export function substrContext(input: string, token: ExpandedLocation): string {
             l,
             sLine === n ? token.startColumn - 1 : 0,
             eLine === n ? token.endColumn : l.length,
-            (s) => s.red.underline,
+            (s) => s,
           )
         }
 
-        return ("" + n).padStart(5).green + " " + l
+        return ("" + n).padStart(5) + " " + l
       })
       .join("\n")
   )
